styles.css

```
@@ -0,0 +1,192 @@
+body {
+    margin: 0 auto;
+    padding: 0;
+    font-family: 'Roboto', sans-serif;
+    display: flex;
+    flex-direction: column;
+    text-align: center;
+    color: #0C0C0C;
+    align-content: center;
+    justify-content: center;
+}
+
+h1, h2, h3, h4, h5, h6 {
+    font-family: 'Merriweather', serif;
+}
+h1{
+    font-size: 50px;
+}
+header{
+    color:white;
+    background-color: #4845E4;
+    display: flex;
+    flex-direction: column;
+    justify-content: center;
+    align-items: center;
+}
+.start{
+    margin: 10% auto;
+}
+footer {
+    text-align: center;
+    color: white;
+    background-color: #0C0C0C;
+    height: 50px;
+    justify-content: center;
+    align-content: center;
+}
+main{
+    display: flex;
+    flex-direction: column;
+    justify-content: center;
+    align-items: center;
+}
+nav{
+    align-self: flex-end;
+    display: flex;
+    justify-content: space-between;
+    flex-direction: row;
+}
+nav p {
+    margin-right: 40px
+}
+button{
+    background-color: #ED4385;
+    color: white;
+    padding: 25px 50px;
+    border-radius: 50px;
+    border: transparent;
+}
+.intro{
+    color: #5722CD;
+    padding: 1px 25%;
+}
+.intro2{
+    color: #5722CD;
+}
+.button2{
+    padding: 25px 80px;
+}
+.flexbox{
+    display: flex;
+    flex-wrap: wrap;
+    background-color: #F3F6F9;
+    flex-direction: column;
+    width: 100%;
+}
+.flexbox2{
+    display: flex;
+    flex-wrap: wrap;
+    justify-content: space-between;
+}
+article{
+    flex: 1 0 33%;
+    margin: 50px;
+    display: flex;
+}
+.infoblok{
+    padding: 10%;
+    align-content: center;
+    align-items: center;
+    text-align: left;
+}
+span{
+    flex: 1;
+    display: flex;
+    justify-content: center;
+    align-items: center;
+}
+
+.infoplaatje{
+    filter: drop-shadow(1px 10px 14px #7A7A7A);
+}
```

Als je hier de width had geset, dan had de afbeelding netjes geschaald als je de pagina smaller maakt
```suggestion
    width: 100%;
}
```